src/components/StampOverlay/StampOverlay.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { withTranslation } from 'react-i18next';
+import onClickOutside from 'react-onclickoutside';
+import getClassName from 'helpers/getClassName';
+import getToolStylePopupPositionBasedOn from 'helpers/getToolStylePopupPositionBasedOn';
+import actions from 'actions';
+import selectors from 'selectors';
+import core from 'core';
+
+import './StampOverlay.scss';
+
+const TOOL_NAME = 'AnnotationCreateRubberStamp';
+const canvasWidth = 160;
+const canvasHeight = 58;
+
+
+class StampOverlay extends React.Component {
+  static propTypes = {
+    activeToolName: PropTypes.string,
+    isDisabled: PropTypes.bool,
+    isOpen: PropTypes.bool,
+    t: PropTypes.func.isRequired,
+    i18n: PropTypes.any,
+    toolButtonObjects: PropTypes.object.isRequired,
+    openElement: PropTypes.func.isRequired,
+    closeElement: PropTypes.func.isRequired,
+    closeElements: PropTypes.func.isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+    this.overlay = React.createRef();
+
+    const { toolButtonObjects } = props;
+    const dataElement = toolButtonObjects[TOOL_NAME].dataElement;
```

Comment:
We can hardcode this but I guess accessing it at runtime is a better practice? :)